compiler/rustc_const_eval/src/const_eval/error.rs

```diff
@@ -7,6 +7,7 @@ use rustc_middle::mir::AssertKind;
 use rustc_middle::query::TyCtxtAt;
 use rustc_middle::ty::TyCtxt;
 use rustc_middle::ty::{layout::LayoutError, ConstInt};
+use rustc_middle::{err_inval, err_machine_stop};
```

Comment:
These are interpreter macros. If we `use` them from a path, then ideally we use them from the `interpret` module, i.e as part o the `use crate::interpret::...` below (based on `crate::interpret` doing a re-export of `rustc_middle::interpret`).

Does that work, or is it really needed to get these from the crate root?

Reply:
I have added another commit that adds a `pub use foo` for each of these macros, and then they can be imported with `use crate::interpret::foo`. I think that's the best way to achieve that.

Reply:
Ah, is `pub use` how one gets an item for a `macro_rules` as the current path? Nice, TIL.